### PR TITLE
[improvement](load) reduce memory in batch for small load channels

### PR DESCRIPTION
**Pull request by zhannngchen**
# Proposed changes

Issue Number: close #xxx

## Problem summary

When there are lots of small load channels, reduce memory in batch to release memory more effectively.

separate some simple optimization to another PR #14215, it's much easier to cherry-pick

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [ ] No
    - [ ] I don't know
3. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
4. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [ ] No Need
5. Does it need to update dependencies:
    - [ ] Yes
    - [ ] No
6. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.49 seconds
 load time: 439 seconds
 storage size: 17178952190 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221111201034_clickbench_pr_44662.html

File: be/src/runtime/load_channel_mgr.h [hunk 1]
```diff
@@ -171,77 +178,115 @@ Status LoadChannelMgr::_handle_mem_exceed_limit(TabletWriterAddResult* response)
...
-    // No matter soft limit or hard limit reached, only 1 thread will wait here,
-    // if hard limit reached, other threads will pend at the beginning of this
-    // method.
-    Status st = channel->handle_mem_exceed_limit(response);
-    LOG(INFO) << "reduce memory of " << *channel << " finished";
+    Status st = Status::OK();
+    for (auto ch : channels_to_reduce_mem) {
+        uint64_t begin = GetCurrentTimeMicros();
+        int64_t mem_usage = ch->mem_consumption();
+        st = ch->handle_mem_exceed_limit(response);
```
Comment (liaoxin01) [inline be/src/runtime/load_channel_mgr.h:285]: Only the last `st` is returned.

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by dataroaring** [comment]
A problem that stopped the stress tests, so merge it quickly.

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Pull request merged**